server/@types/viewModels/index.d.ts

```diff
@@ -69,6 +69,11 @@ declare module 'viewModels' {
     problemRetrievingData: boolean
   }
 
+  export interface GoalsOrProblem {
```

Comment:
I'm not keen on this `OrProblem` terminology here. I know you're taking it from what we've started to introduce in the API codebase, but that terminology is part of the inner implementation detail of the API codebase and not exposed in the swagger spec or similar.
I don't think I want to introduce that terminology here in the UI - we've not used it elsewhere

Following the patterns of terminology already used, is it not simply `Goals`? (or have we already used that for something else?)

Reply:
Well it's actually because it's quite literally `goals` or `problemRetrievingData` which is not terminology I introduced and writing `goals.goals` feels bad. I can rename it to Goals but it's super painful if I split the PR.